Cargo.toml

```diff
@@ -17,6 +17,9 @@ keywords = [
 ]
 categories = ["development-tools::debugging", "development-tools::profiling"]
 
+[features]
+resource_selector = []
```

Comment:
I know the readme was shouting a lot about performance, but honestly it's fine to have it by default, 1% perf hit is fine!